### PR TITLE
Update pi_hole docs for 0.99

### PR DESCRIPTION
**Pull request by johnluetke**
**Description:**

Clarifying Pi-hole documentation per home-assistant/home-assistant#26806

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by johnluetke** [comment]
There have been some forum posts and github issues filed around changes to the pi-hole integration, this PR attempts to clarify the documentation a bit more.

File: source/_components/pi_hole.markdown [hunk 1]
```diff
@@ -49,7 +54,10 @@ verify_ssl:
...
 # Example configuration.yaml entry
 pi_hole:
-  host: IP_ADDRESS
+  host: 'localhost:4685'
+  location: admin
+  ssl: false
+  verify_ssl: false
 ```
-
+---
```
Comment (fabaff) [inline source/_components/pi_hole.markdown]: Remove this please.

File: source/_components/pi_hole.markdown [hunk 2]
```diff
@@ -23,10 +23,15 @@ pi_hole:
 
 {% configuration %}
 host:
-  description: IP address of the host where Pi-hole is running.
+  description: >
+    Hostname [and port] of the host where Pi-hole is running.
+
+
+    **Note:** If your Pi-Hole instance is the Hass.io add-on, you *must* specify port `4685`
```
Comment (fabaff) [inline source/_components/pi_hole.markdown]:
```suggestion
    **Note:** If your Pi-Hole instance is the Hass.io add-on, you *must* specify port `4685`.
```

File: source/_components/pi_hole.markdown [hunk 3]
```diff
@@ -23,10 +23,15 @@ pi_hole:
 
 {% configuration %}
 host:
-  description: IP address of the host where Pi-hole is running.
+  description: >
+    Hostname [and port] of the host where Pi-hole is running.
```
Comment (fabaff) [inline source/_components/pi_hole.markdown]: We should show how the entry has to look.

```suggestion
    The hostname (and port), e.g., '192.168.0.3:4685' of the host where Pi-hole is running.
```

**Review by fabaff — DISMISSED** [review]

File: source/_components/pi_hole.markdown [hunk 4]
```diff
@@ -49,7 +54,10 @@ verify_ssl:
 ```yaml
 # Example configuration.yaml entry
 pi_hole:
-  host: IP_ADDRESS
+  host: 'localhost:4685'
+  location: admin
```
Comment (frenck) [inline source/_components/pi_hole.markdown]: Admin location is the default and is not needed to be specified.

File: source/_components/pi_hole.markdown [hunk 5]
```diff
@@ -10,11 +10,11 @@ redirect_from:
  - /components/sensor.pi_hole/
 ---
 
-The `pi_hole` integration allows you to retrieve and display statistics from a [Pi-hole](https://pi-hole.net/) system.
+The `pi_hole` integration allows you to retrieve and display statistics from a single [Pi-hole](https://pi-hole.net/) system.
 
 ## Configuration
 
-To enable this sensor, add the following lines to your `configuration.yaml` file for a GET request:
+To enable this platform with the default configuration, add the following lines to your `configuration.yaml` file
```
Comment (frenck) [inline source/_components/pi_hole.markdown]:
```suggestion
To enable this integration with the default configuration, add the following lines to your `configuration.yaml` file
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @johnluetke! 👍

**Pull request merged**